assignment/scripts/array-practice.js

```diff
@@ -9,37 +9,40 @@ console.log('Animals are: ', animalArray);
 
 // 1.a. TODO: Make an array with some favorite foods
 
+let foodsArray = ['sushi', 'empanadas', 'frech fries', 'hamburgers'];
 
 // 1.b. TODO: Log your array of foods to the console with a message, similar 
 //      to the example above
 
-
+console.log('My favorite foods are: ', foodsArray);
 
 // 2. Array.length - tells you how many items are in the array
-console.log('--- 2. Length of an array ---');
+console.log('number of favorite foods:', foodsArray.length);
 
 // Example: How many animals are in the array? 
 console.log('Number of animals:', animalArray.length);
 
 // 2.a. TODO: Log to the console the number of foods in your array
 
+console.log('My top foods are: ', foodsArray.length);
 
 // 3. Accessing array items
-console.log('--- 3. Accessing items in an array ---');
+console.log('My first choice is: ', foodsArray[0]);
 
 // Example: Log the first animal from the array using it's array index
-console.log('First animal is', animalArray[0])
+console.log('First animal is', animalArray[0]);
 
 // 3.a. TODO: Log the second animal in the array 
 
-
+console.log('Second animal is', animalArray[1]);
 // 3.b. TODO: Log the last animal in the array using it's array index 
 
+console.log('Fourth animal is', animalArray[3]);
 
 // 3.c. (STRETCH) TODO: Log the last animal by using the array length, 
 //      instead of the exact index number of the last item
 
-
+console.log('last animal' ,[animalArray.length -1]);
```

Comment:
Not quite. I think you maybe meant to `animalArray` in front of the square brackets.